### PR TITLE
Hp/event page links without js

### PR DESCRIPTION
**Pull request by hannahpurcell**
#### Summary of changes
**Asana Ticket:** [With js disabled, "Jump to"' links don't do anything (Firefox on Mac)](https://app.asana.com/0/555089885850811/1200108181856603)

Left-hand links all add a "#" to the url that helps the page scroll even when JS is disabled.  When JS is disabled, the nav dropdowns are non-functional, so those have been hidden for now.

---

Before getting review, please check the following:

* [x] Does frontend functionality render and work correctly in IE?  **Unfortunately cannot test disabled JS because BrowserStack won't let me disable it...  but enabled JS looks fine**
* [ ] Are interactive elements accessible to screen readers?

### CODE REVIEW

File: apps/site/lib/site_web/views/event_view.ex [hunk 1]
```diff
@@ -169,4 +169,16 @@ defmodule SiteWeb.EventView do
...
+  def build_nav_path(conn, month, year, toggle) do
+    view_is_calendar = conn.assigns[:calendar_view]
+    if (view_is_calendar && not toggle) or (not view_is_calendar && toggle) do
+      # calendar view
+      event_path(conn, :index, [month: month, year: year, calendar: true])
+    else
+      # list view
+      "#{event_path(conn, :index, [month: month, year: year])}##{month}-#{year}"
+    end
+  end
```
Comment (alicia-mbta) [inline apps/site/lib/site_web/views/event_view.ex:183]: Thanks for the clean refactor 🙏

**Review by alicia-mbta — CHANGES_REQUESTED** [review]

File: apps/site/lib/site_web/templates/event/_year_select.html.eex [hunk 2]
```diff
@@ -5,7 +5,7 @@
       :option,
       year,
       [
-        value: event_path(@conn, :index, month: 1, year: year),
+        value: build_nav_path(@conn, @month, year),
```
[the remark this answers is not in the document]
Comment (hannahpurcell) [reply apps/site/lib/site_web/templates/event/_year_select.html.eex:8]: Oh ok!  I expected the other way around but understand why starting at the beginning of the year would make sense.  I'll update it 👍

**Review by thecristen — COMMENTED** [review]
Just a comment to add to Alicia's! :)

File: apps/site/assets/css/_events.scss [hunk 3]
```diff
@@ -164,6 +164,10 @@ $mobile-control-height: 73px;
   border-radius: 4px;
   border-width: 1px;
   padding: .5 * $base-spacing 2 * $base-spacing .5 * $base-spacing $base-spacing;
+  display: none;
+  &.js {
```
[the remark this answers is not in the document]
Comment (hannahpurcell) [reply apps/site/assets/css/_events.scss]: Oh so cool!!

**Review by alicia-mbta — APPROVED** [review]

**Review by thecristen — APPROVED** [review]

**Pull request merged**